Objective-C/CBLDatabase.mm

```diff
@@ -527,8 +552,6 @@ - (void) removeChangeListenerWithToken: (id<CBLListenerToken>)token {
     CBLAssertNotNil(token);
     
     CBL_LOCK(self) {
-        [self mustBeOpen];
```

Comment:
What relevance does this have to the PR?

Reply:
Good catch. It's a mistake. Not sure why it got removed.

Reply:
Actually I removed it because it doesn't need it.